doc/playground.lua

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env tarantool
+
+-- How to run:
+--
+-- $ ./doc/playground.lua
+--
+-- Or
+--
+-- $ KEEP_DATA=1 ./doc/playground.lua
+--
+-- What to do next:
+--
+-- Choose an example from doc/select.md or doc/pairs.md and run.
+-- For example:
+--
+-- tarantool> crud.select('developers', nil, {first = 6})
+--
+-- You can also use help() and prev() functions to navigate over
+-- examples in the documentation.
+
+local fio = require('fio')
+local vshard = require('vshard')
+local crud = require('crud')
+local console = require('console')
+
+-- Trick to don't leave *.snap, *.xlog files. See
+-- test/tuple_keydef.test.lua in the tuple-keydef module.
+if os.getenv('KEEP_DATA') ~= nil then
+    box.cfg()
+else
+    local tempdir = fio.tempdir()
+    box.cfg({
```

Comment:
I'm not clear with https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#idioms-and-patterns

Is box.cfg{} necessary or box.cfg({}) is acceptable?

Reply:
I have to consider both variants as acceptable. To be honest, if I would write this style guide now, I would not recommend `func{<...>}` form for any function, because this way it is not perfectly clear what function should be called this way and what shouldn't.

Reply:
Thank you!